docroot/themes/humsci/su_humsci_gin_admin/su_humsci_gin_admin.libraries.yml

```diff
@@ -13,3 +13,23 @@ login_portal:
 fix-autocomplete-width:
   js:
     js/fix-autocomplete-width.js: {}
+
+humsci-traditional-styling:
+  css:
+    theme:
+      ../humsci_traditional/css/humsci_traditional-preview.css: {}
+  js:
+    ../humsci_traditional/js/index.js: {}
+
+humsci-colorful-styling:
+  css:
+    theme:
+      ../humsci_colorful/css/humsci_colorful-preview.css: {}
+  js:
+    ../humsci_colorful/js/index.js: {}
```

Comment:
These libraries should be added to the relevant themes instead of referencing file paths outside of the scope of the theme. 

For example, I'd create `admin-preview` libraries that include the `prevew.css` and `index.js` files in the `humsci_colorful.libraries.info.yml` in the `humsci_colorful` theme itself (and the same for `humsci_traditional`). Then you could include them programmatically with a hook.

Reply:
@cienvaras Just want to make sure you've seen these.

Reply:
That's actually even better 👍 Thanks !